modules/oauth-default-clients.adoc

```diff
@@ -18,6 +18,10 @@ The following OAuth clients are automatically created when starting the {product
 |`openshift-challenging-client`
 |Requests tokens with a user-agent that can handle `WWW-Authenticate` challenges.
 
+|'openshift-cli-client'
+| Requests tokens with a user-agent using an authorization code via the web browser.
```

Comment:
```suggestion
| Requests tokens by using a localhost HTTP server fetching an authorization code grant.
```

Reply:
@stlaz did you maybe mean "local HTTP server"?

Reply:
@stlaz did you maybe mean "local HTTP server"?

Reply:
either is fine with me